DragonFruit.Six.API/Stats/SeasonalStats.cs

```diff
@@ -39,6 +48,31 @@ public class Season
         [JsonProperty("maxmmr")]
         public double MaxMMR { get; set; }
 
+        [JsonProperty("next_rank_mmr")]
```

Comment:
These property headers aren't used on ubi's side so if you want to name them something easier to understand then feel free. These will probably be used in a backend if it were the case.

Reply:
I think it is good to keep the headers consistent with Ubisoft's side. I will let you decide however.

Reply:
we can leave these as they are. They aren't as bad as some of the stats ones, like the general ones.